### ISSUE TITLE
Alternate fix minimums not captured

### ISSUE THREAD

**Issue by andrewda** [ammaraskar/faa-instrument-approach-db#3]
Some approaches (e.g. https://aeronav.faa.gov/d-tpp/2407/00330IL10R.PDF) include separate minimums for a particular type when a specific fix can be identified. 

I believe I've seen approaches with multiple "__ FIX MINIMUMS" sections as well, but I'm not able to find any examples right now.

![CleanShot 2024-07-31 at 08 31 54@2x](https://github.com/user-attachments/assets/906fb6e1-66ac-477e-8df5-73e1eda6b576)

**Comment by ammaraskar** [comment]
Yup, this would be good to capture. I currently skipped it but left a field for it in the text extractor https://github.com/ammaraskar/faa-instrument-approach-db/blob/d4c8302da0065763a85d589628d1b1302e4cfe72/plate_analyzer/text_extraction.py#L35-L37

We can likely encode that condition field in the minimums json as well.
